ray-operator/config/samples/pytorch-text-classifier/ray-job.pytorch-distributed-training.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  generateName: pytorch-text-classifier-
+spec:
+  entrypoint: python /home/ray/kuberay/kuberay-master/ray-operator/config/samples/pytorch-text-classifier/fine-tune-pytorch-text-classifier.py
+  runtimeEnvYAML: |
+    pip:
+      - numpy
+      - datasets
+      - transformers>=4.19.1
+      - pytorch-lightning==1.6.5
+  rayClusterSpec:
+    rayVersion: '2.9.0'
+    headGroupSpec:
+      rayStartParams:
+        dashboard-host: '0.0.0.0'
+      template:
+        spec:
+          initContainers:
+          - name: download-kuberay
+            image: busybox
+            command: ["/bin/sh", "-c"]
+            args:
+              - wget https://github.com/ray-project/kuberay/archive/master.zip && unzip master.zip -d /home/ray/kuberay && ls /home/ray/kuberay/kuberay-master
+            volumeMounts:
+            - name: kuberay-repo
+              mountPath: /home/ray/kuberay
+          containers:
+            - name: ray-head
```

Comment:
This RayJob uses a single-node Ray cluster to avoid configuring shared storage for checkpointing. Otherwise I would need to configure something like GCSFuse which would not work on every environment